nuxeo-core/nuxeo-core-convert/src/main/java/org/nuxeo/ecm/core/convert/service/ConversionWork.java

```diff
@@ -61,8 +71,29 @@ public ConversionWork(String converterName, String destinationMimeType, BlobHold
     }
 
     protected void storeInputBlobHolder(BlobHolder blobHolder) {
-        inputEntryKey = entryKey + "_input";
-        putBlobHolder(inputEntryKey, blobHolder);
+        if (!storeManagedBlob(blobHolder)) {
+            // standard conversion
+            inputEntryKey = entryKey + "_input";
+            putBlobHolder(inputEntryKey, blobHolder);
+        }
+    }
+
+    /**
+     * @since 10.1
+     */
+    protected boolean storeManagedBlob(BlobHolder blobHolder) {
+        Blob blob = blobHolder.getBlob();
+        if (!(blob instanceof ManagedBlob) || blobHolder.getBlobs().size() > 1) {
```

Comment:
As a precaution, please also check add `|| !(blob instanceof Serializable)` otherwise we won't be able to serialize the `ConversionWork`.